### PR TITLE
fix: remove indentation from lists in Feature descriptions

### PR DESCRIPTION
**Pull request by christian-dinh**
### Overview

Dev says it looks good and fixes the spacing issues brought up in QA today, so 🎉 

Made the change for both `ul` and `ol`

Screen Shot 2021-02-08 at 3 00 06 PM

### PR Checklist

- [ ] Related to designs:
- [x] Related to JIRA ticket: [REACH-598](https://codecademy.atlassian.net/browse/REACH-598)
- [x] I have run this code to verify it works
- [ ] This PR includes unit tests for the code change

### CODE REVIEW

File: packages/gamut-labs/src/landingPage/Feature.tsx [hunk 1]
```diff
@@ -54,6 +54,14 @@ export const FeaturedTitle: React.FC<FeaturedTitleProps> = ({
   </Text>
 );
 
+const List = styled.ul`
+  padding: 0;
+  list-style-position: inside;
```
Comment (rogervera) [inline packages/gamut-labs/src/landingPage/Feature.tsx:59]: TIL https://developer.mozilla.org/en-US/docs/Web/CSS/list-style-position

Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/Feature.tsx:59]: same lol

Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/Feature.tsx:59]: actually reading that I'm gonna see if I can just put it on `li`s instead

File: packages/gamut-labs/src/landingPage/Feature.tsx [hunk 2]
```diff
@@ -63,7 +71,19 @@ export const FeaturedDescription: React.FC<FeaturedDescriptionProps> = ({
   onAnchorClick,
 }) => (
   <Box textColor="navy">
-    <Markdown text={desc} spacing="none" onAnchorClick={onAnchorClick} />
+    <Markdown
+      text={desc}
+      spacing="none"
+      onAnchorClick={onAnchorClick}
+      overrides={{
+        ul: {
```
Comment (judahtanthony) [inline packages/gamut-labs/src/landingPage/Feature.tsx:79]: Was this preferable to just using [Nested Selectors](https://emotion.sh/docs/nested)?

Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/Feature.tsx:79]: wasn't sure, I did that first and then discovered `overrides` which seemed less reliant on Markdown's implementation

Comment (rogervera) [reply packages/gamut-labs/src/landingPage/Feature.tsx:79]: Hm, good point Judah. I'm for just going with nested selectors too then. The overrides are just for CSS needs anyway.

Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/Feature.tsx:79]: >wasn't sure, I did that first and then discovered overrides which seemed less reliant on Markdown's implementation

To clarify, using overrides means we don't have to care about `Markdown` adding its own styles to different elements. Doubt that would happen, but it is built around [marked](https://www.npmjs.com/package/marked) which we have no control over 🤷

Comment (christian-dinh) [reply packages/gamut-labs/src/landingPage/Feature.tsx:79]: but yea I'll switch to nested selectors

**Comment by codecademydev** [comment]
<h3>📬Published Alpha Packages:</h3><pre>@codecademy/gamut-labs@9.2.5-alpha.7adb14.0<br/>@codecademy/styleguide@25.1.6-alpha.7adb14.0<br/></pre>

**Comment by codecademydev** [comment]
<p>🚀 Styleguide deploy preview ready!</p><p><a href=https://60219e1d7574fa00a5d2f92b--gamut-preview.netlify.app>https://60219e1d7574fa00a5d2f92b--gamut-preview.netlify.app</a></p><p><a href=https://app.netlify.com/sites/gamut-preview/deploys/60219e1d7574fa00a5d2f92b>Deploy Logs</a></p>

**Review by aionate0812 — APPROVED** [review]

**Review by rogervera — APPROVED** [review]
:shipit:

**Pull request merged**